packages/ember-metal/tests/utils_test.js

```diff
@@ -10,6 +10,7 @@ QUnit.module('Ember Metal Utils');
 QUnit.test('inspect outputs the toString() representation of Symbols', function() {
   // Symbol is not defined on pre-ES2015 runtimes, so this let's us safely test
   // for it's existence (where a simple `if (Symbol)` would ReferenceError)
+  /* jshint latedef: false */
   let Symbol = Symbol || null;
```

Comment:
the following very much fails:

```js
(function() { let Symbol = Symbol || null;}());
```

with 
```
VM257:1 Uncaught ReferenceError: Symbol is not defined
```

`let Symbol` ensures that before that declaration `Symbol` is in TDZ. basically, it is non-addressable until after the `let`.  As it turns out, the right hand side of `let Symbol = <right hand side>` is technically before the `let`, and since `let` has reserved `Symbol` for that scope, even if the a global `Symbol` exists, this will error.